samples/crypto/aes_gcm/sample.yaml

```diff
@@ -4,8 +4,8 @@ sample:
     name: AES GCM example
 tests:
     sample.aes_gcm:
-        tags: introduction
-        platform_allow: nrf5340dk_nrf5340_cpuapp_ns nrf5340dk_nrf5340_cpuapp nrf9160dk_nrf9160_ns nrf9160dk_nrf9160 nrf52840dk_nrf52840
+        tags: introduction psa cc3xx
+        platform_allow: nrf5340dk_nrf5340_cpuapp_ns nrf5340dk_nrf5340_cpuapp nrf9160dk_nrf9160_ns nrf9160dk_nrf9160
```

Comment:
The 9160 has oberon for GCM since it has the same CryptoCell as the 52840. Please add the 9160 the same way that you added the 52840.

Reply:
Ah ok!
So the split is based on cc310 vs cc312.

I force pushed a change to make that the split instead.